txpipe/twopoint.py

```diff
@@ -389,7 +454,17 @@ def call_treecorr(self, i, j, k):
         This is a wrapper for interaction with treecorr.
         """
         import sacc
-
+        import pickle
```

Comment:
Can you add a `TODO` here for me to fix up the caching code?

Reply:
Done.